spacialgaze-plugins/economy.js

```diff
@@ -321,11 +322,12 @@ exports.commands = {
 		this.sendReply('Your symbol has been removed.');
 	},
 
+	economy: 'economystats',
 	currency: 'economystats',
 	stardust: 'economystats',
 	economystats: function (target, room, user) {
 		if (!this.runBroadcast()) return;
-		const users = Object.keys(Db.currency.object());
+		const users = Object.keys(Db.currency.object);
```

Comment:
Dosent exists, try modifying it to work with `Db.currency.keys()` which is an array of every entry in the db (all userids in this case)